.development/googlesheets_dataclean.R

```diff
@@ -0,0 +1,40 @@
+library(tidyverse)
+library(googlesheets4)
+
+sheet_url <- "https://docs.google.com/spreadsheets/d/1usk9Q-5lA4bL_z6KXpUohc_2x_KhDgLxtm-YEtim_yk/"
+
+# define google sheets ----------------------------------------------------
+## may consider using purrr once we get to more sheets
+
+data_master_cnw <- googlesheets4::read_sheet(ss = sheet_url, sheet = "Central & Western")
+data_master_wanchai <- googlesheets4::read_sheet(ss = sheet_url, sheet = "Wan Chai")
+data_master_eastern <- googlesheets4::read_sheet(ss = sheet_url, sheet = "Eastern")
+data_master_southern <- googlesheets4::read_sheet(ss = sheet_url, sheet = "Southern")
+
+
+# clean googlesheets ------------------------------------------------------
+
+clean_gsheet_dat <- function(data){
+  data %>%
+    janitor::clean_names() %>%
```

Comment:
If you did `clean_names()` instead, would you get the same results?

Reply:
Just curious as to why you need to prepend `clean_names()` since there doesn't seem to be a conflict in multiple functions being called this.